tests/client/gridfs.rs

```diff
@@ -0,0 +1,105 @@
+use bson::Bson;
+
+use mongodb::{Client, ThreadedClient};
+use mongodb::coll::options::FindOptions;
+use mongodb::db::ThreadedDatabase;
+use mongodb::gridfs::{Store, ThreadedStore};
+use mongodb::gridfs::file::DEFAULT_CHUNK_SIZE;
+
+use rand::{thread_rng, Rng};
+use std::io::{Read, Write};
+
+#[test]
```

Comment:
Awesome test. But I'd like to see some test cases for:
- remove_id/remove
- find